composer.json

```diff
@@ -57,6 +57,7 @@
     "require-dev": {
         "phpunit/phpunit": "^8.5",
         "zircote/swagger-php": "^3.1",
-        "fakerphp/faker": "^1.13"
+        "fakerphp/faker": "^1.13",
+        "osteel/openapi-httpfoundation-testing": "^0.8.0"
```

Comment:
It looks like the most recent available version is 0.10.2. It looks like by 0.9.0 they dropped support for PHP 7.2, so I'm guessing that's why we're using 0.8.0?

Reply:
Likely I just did `composer require --dev osteel/openapi-httpfoundation-testing` and let the system decide.